### PR TITLE
Radiobutton design fix

### PR DESCRIPTION
**Pull request by libertymayc**
Minor fix noticed by @pseys 
Not a huge deal - but the actual control in the RadioButton doesn't match the size of the icon presented

### CODE REVIEW

**Review by joshwooding — CHANGES_REQUESTED** [review]
I believe this breaks the Checkbox alignment.

**Comment by joshwooding** [comment]
> > I believe this breaks the Checkbox alignment.
> 
> how so?

Applying align-items to Checkbox means that when there is a long label the checkbox aligns to the centre rather than the baseline.

**Review by joshwooding — CHANGES_REQUESTED** [review]
What issue is this solving, can you raise a bug

File: .changeset/soft-ligers-run.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/core": patch
+---
+
+Fixed control size in RadioButton to match the shown icon size
```
Comment (joshwooding) [inline .changeset/soft-ligers-run.md:5]:
```suggestion
Fixed control size in Radio Button to match the shown icon size.
```

**Review by joshwooding — APPROVED** [review]

**Review by joshwooding — APPROVED** [review]

**Review by Fercas123 — APPROVED** [review]

**Pull request merged**